lib/src/component_declaration/component_base.dart

```diff
@@ -175,7 +166,10 @@ abstract class UiComponent<TProps extends UiProps> extends react.Component {
   @override
   TProps get props {
     var unwrappedProps = this.unwrappedProps;
-    TProps typedProps = _typedPropsCache[unwrappedProps];
+    /// Have to cast as [TProps] until we can parameterize [_typedPropsCache].
```

Comment:
@greglittlefield-wf @jacehensley-wf `Expando`s parameterized, casts removed in https://github.com/Workiva/over_react/pull/15/commits/c8b41b7bffb83b930102c2f10586314704dbeb3c